examples/add_classification.py

```diff
@@ -0,0 +1,66 @@
+import pdb
+from zegami_sdk.client import ZegamiClient
+from zegami_sdk import nodes
+
+WORKSPACE_ID = ''
+COLLECTION_ID = ''
+
+
+zc = ZegamiClient()
+
+
+workspace = zc.get_workspace_by_id(WORKSPACE_ID)
+# import pdb; pdb.set_trace()
```

Comment:
I know it's just an example but might be worth cleaning

Reply:
Worth thinking a little about the future of these 'examples'. I've also snuck another into this PR to add clustering, which I created yesterday.
In reality these are actual tools we'll be using on collections, rather than mere examples. Perhaps they belong in a separate repo, or maybe only if they're customer-specific.